canonical-websites/insights.ubuntu.com
# Cache on disk

In production, the app was caching in memory
(`REQUEST_CACHE_BACKEND` wasn't defined), leading to the production
application constantly hitting its memory limit of 256M leading to
frequent OOMkills.

We should always simply cache on disk instead.

Fixes #479

QA
--

`./run`, browse around the site a bit, check `cache.sqlite` is updated as expected.
